content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,488 @@
+---
+layout: engineering-education
+status: publish
+published: true 
+url: /statistical-hypothesis-testing-python-implementation/
+title: Statistical Hypothesis Testing With Python Implementation
+description: This tutorial is a comprehensive explanation and implementation of statistical hypothesis testing methods such as t-test, chi-squared test & ANOVA in python, It is well known that most of the implementation of these techniques are always in R, and most Data Scientists and Analysts who use python always shy away from it.
```

Comment:
Remove the second line and punctuate t-test as "t-Test" and so forth.

Reply:
When you say second line, do you mean the url line?

Reply:
I meant the second line in the description tag.